pkg/api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
Not sure, but possibly because I use Go 1.17 locally. Even on controller-gen, the [commit](https://github.com/kubernetes-sigs/controller-tools/commit/44db28fcac75b95dbca2ffafb440b0ff02f47c70) which they bumped to Go 1.17 has this line on all autogenerated files. I will remove it, since we use Go 1.16 for the CLI.

Reply:
Thanks for calling it out 🤛